CHANGELOG.md

```diff
@@ -18,15 +19,15 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
   * AKS version [1.30](https://kubernetes.io/blog/2024/04/17/kubernetes-v1-30-release/) is available in preview. 
     
  * Bug Fixes:
-  * CoreDNS has been updated to use image [v1.9.4-hotfix.20240520](https://github.com/aks-lts/coredns/tree/release-1.9) on all AKS clusters above version 1.24. This updated image addresses [CVE vulnerabilities](https://github.com/aks-lts/coredns/commit/ba698d28c2ab8d9db0951592be631885e4134e5).
```

Comment:
This was just fully removed? What's the answer?

Reply:
I ll check with CCP for a better explanation otherwise omitting it for now.